cmd/nodecmd/create.go

```diff
@@ -137,8 +137,15 @@ will apply to all nodes in the cluster`,
 func handlePostRun(_ *cobra.Command, _ []string) {}
 
 func preCreateChecks(clusterName string) error {
-	if !flags.EnsureMutuallyExclusive([]bool{useLatestAvalanchegoReleaseVersion, useLatestAvalanchegoPreReleaseVersion, useAvalanchegoVersionFromSubnet != "", useCustomAvalanchegoVersion != ""}) {
-		return fmt.Errorf("latest avalanchego released version, latest avalanchego pre-released version, custom avalanchego version and avalanchego version based on given subnet, are mutually exclusive options")
+	if useCustomAvalanchegoVersion != "" || useAvalanchegoVersionFromSubnet != "" {
+		useLatestAvalanchegoReleaseVersion = false
+		useLatestAvalanchegoPreReleaseVersion = false
+	}
+	if !flags.EnsureMutuallyExclusive([]bool{useLatestAvalanchegoReleaseVersion, useLatestAvalanchegoPreReleaseVersion}) {
```

Comment:
why do we allow users to use useLatestAvalanchegoReleaseVersion and useCustomAvalanchegoVersion at the same time ?

Reply:
pls ignore comment above. We need to tell users that `useLatestAvalanchegoReleaseVersion` and `useLatestAvalanchegoPreReleaseVersion` have lower priority than `useLatestAvalanchegoReleaseVersion` and `useCustomAvalanchegoVersion` and will be override.